tools/homebrew-start/lanraragi

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+trap "kill 0" SIGINT
+
+# prefix/place where lrr is installed
+workdir=$(dirname $0)/$(dirname $(readlink $0))/..
+
+if [[ ${OSTYPE} == "darwin"* ]]; then
+  _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
+else
+  _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
+fi
+# modify redis config
+sed -i "" "s/<LRR_DATA_DIRECTORY>/${_LRR_DATA_DIRECTORY}/g" ${workdir}/redis.conf
```

Comment:
Using / as the delimiter can fail on some seds if the DATA_DIRECTORY variable contains forward slashes as well.
Replacing it with ~ seems to finally remove the sed errors on my end. 
```suggestion
echo "Data Directory set to ${_LRR_DATA_DIRECTORY}"
sed -i "" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
```

Reply:
Ah yeah, that’s like the most common error with sed. I do not use it very often but when I do, I forget about that :/